docs/manual/docs/cmd/outlook/report/report-getmailboxusagestorage.md

```diff
@@ -0,0 +1,40 @@
+# outlook report getmailboxusagestorage
+
+Get the amount of storage used in your organization. 
```

Comment:
Could you please add a `mailbox` in the sentence? For me, it was not clear what kind of storage is about when I first opened the PR.